### ISSUE TITLE
False positive: Secure Cookie Transfer [PHP7.3 options array usage]

### ISSUE THREAD

**Issue by rbairwell** [kalessil/phpinspectionsea#1378]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         |  Php Inspections (EA Ultimate) 3.0.2 |
| Language level | PHP 7.4beta                      |

Current behaviour
--
The following code
```
<?php
$options=['secure'=>true,'httponly'=>true];
setcookie('name','value',$options);
```
triggers the 'Secure Cookie Transfer' inspection warning.

Expected behaviour
--
Ideally recognise if the third parameter (`options`) is an array on PHP7.3 or above (see https://www.php.net/manual/en/function.setcookie.php#refsect1-function.setcookie-changelog ) and then either not show the warning or, if PHPInspections can check the contents of the array, that that if secure/httponly etc is set.
 
Environment details
---
PhpStorm 2019.2
Build #PS-192.5728.108, built on July 24, 2019
Runtime version: 11.0.3+12-b304.10 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 972M
Cores: 8
Registry: run.processes.with.pty=TRUE
Non-Bundled Plugins: GrepConsole, Key Promoter X, PHPUnit code coverage, String Manipulation, com.intellij.ideolog, BashSupport, com.kalessil.phpStorm.phpInspectionsUltimate, de.espend.idea.php.phpunit, de.espend.idea.php.toolbox, de.espend.idea.php.annotation, izhangzhihao.rainbow.brackets, org.psliwa.idea.composer, ru.adelf.idea.dotenv

**Comment by kalessil** [comment]
Fixed, currently we'll skip analysis for options array. I'll think also if we can efficiently lookup arrays structure - this was never our focus so far.
